packages/jaeger-ui/src/components/TraceDiff/TraceDiffHeader/TraceTimelineLink.test.js

```diff
@@ -13,30 +13,31 @@
 // limitations under the License.
 
 import * as React from 'react';
-import { shallow } from 'enzyme';
+import { render, screen, fireEvent, createEvent } from '@testing-library/react';
 
 import TraceTimelineLink from './TraceTimelineLink';
-import NewWindowIcon from '../../common/NewWindowIcon';
 
 describe('TraceTimelineLink', () => {
   const traceID = 'test-trace-id';
-  let wrapper;
 
   beforeEach(() => {
-    wrapper = shallow(<TraceTimelineLink traceID={traceID} />);
+    render(<TraceTimelineLink traceID={traceID} />);
   });
 
   it('renders the NewWindowIcon', () => {
-    expect(wrapper.find(NewWindowIcon).length).toBe(1);
+    expect(screen.getAllByTestId('NewWindowIcon').length).toBe(1);
   });
 
   it('links to the given trace', () => {
-    expect(wrapper.find('a').prop('href')).toBe(`/trace/${traceID}`);
+    expect(screen.getByRole('link').href).toBe(`http://localhost/trace/${traceID}`);
```

Comment:
```suggestion
    expect(screen.getByRole('link')).toHaveAttribute('href', `/trace/${traceID}`);
```

I think checking for the jsx dom attribute would make more sense in this case rather than the rendered dom attribute.

Reply:
`expect(...).toHaveAttribute is not a function`

but changing to `toHaveProperty` still results in full URL:
```
    expect(received).toHaveProperty(path, value)

    Expected path: "href"

    Expected value: "/trace/test-trace-id"
    Received value: "http://localhost/trace/test-trace-id"
```

Reply:
Yes, you need to import `import '@testing-library/jest-dom';` for it to work